src/analysis/sensitivity.jl

```diff
@@ -3,7 +3,6 @@ module sensitivity
 using Logging
 
 using
-    AxisKeys,
     NamedDims,
```

Comment:
Do you still need to import `NamedDims`?

Reply:
NamedDims is still required for the version of `outcome_map` contained in this pull request. `outcome_map` is migrates in a seperate pull request #702.

Yes it should eventually be removed however I didn't want to break precompilation.